CRM/Report/Form.php

```diff
@@ -2398,6 +2398,11 @@ public function select() {
 
```

Comment:
@eileenmcnaughton instead doing it in IF loop how about changing it here as:
```
     $select = empty($select) ? array('1') : $select;
```

Reply:
@monishdeb Her example was also checking for just "select", which seems like a good idea?

Reply:
@mlutfy wouldn't that only happen if `$select` is empty as per L2400 where $this->_select is constructed by adding "SELECT" onto the front of `$select`

Reply:
I think the revised change is what @monishdeb meant - although I have not squashed it onto one line because I wanted to keep the comment